lib/page/get-display-value/get-display-value.unit.spec.js

```diff
@@ -69,7 +71,125 @@ test('When there is no value', t => {
   const pageInstance = {}
 
   const displayValue = getDisplayValue(pageInstance, userData, nameInstance)
-  t.equals(displayValue, 'Not answered', 'it should return the expected value for unanswered')
+  t.equals(displayValue, 'Not answered', 'it should return \'Not answered\'')
+
+  t.end()
+})
+
+test('When the value is null', t => {
```

Comment:
Coverage looks good.
Is defining things like nameInstance each time required?

Can it be promoted to a higher scope to be reused since it doesn't change?

Reply:
Good point

Reply:
Done